website-page1.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="style.css" rel="stylesheet">
+    <title>Muna Fidow Website</title>
+    <style="background-color: darkolivegreen;"></style>
+  </head>
+  <body>
+    <h1>Muna Fidow Story</h1>
+    <div class="flex-container">
+    <div><img src="images/muna.jpg"WIDTH="300" HEIGHT="300" alt="Muna"/></div>
+    <div><img src="images/goodvibes.jfif"WIDTH="300" HEIGHT="300" alt="good vibes"/></div>
+    </div>
+    <p id="p">I work as an administrator in a  construction company, I like sushi and the colour green. In my spare time I read fiction books, tend to my house plants and watch tv shows. I like Money Heist.
+    <p id="p">I was born and grew up in Italy. I like to vacation where there are bodies of water even though I cant swim. My favourite animal is an Elephant.</p>
```

Comment:
id's should be unique - is it needed here? Could it be better to reference the `p` tag as all instances have the same style